dronecan/sensors/rc/20400.RCInput.uavcan

```diff
@@ -0,0 +1,6 @@
+uint8 STATUS_RSSI_VALID = 1 # RSSI field is valid
+uint8 STATUS_FAILSAFE = 2 # receiver has lost contact with transmitter
+uint16 status
+
+uint8 rssi
```

Comment:
percentage? How to handle the difference between not provided and zero?

Reply:
there is a status bit for provided